staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go

```diff
@@ -971,14 +975,18 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 				sw.Do("out.$.name$ = in.$.name$\n", args)
 				continue
 			}
+			conversionExists := true
 			if g.convertibleOnlyWithinPackage(inMemberType, outMemberType) {
 				sw.Do("if err := "+nameTmpl+"(&in.$.name$, &out.$.name$, s); err != nil {\n", args)
 			} else {
-				sw.Do("// TODO: Inefficient conversion - can we improve it?\n", nil)
-				sw.Do("if err := s.Convert(&in.$.name$, &out.$.name$, 0); err != nil {\n", args)
+				args := argsFromType(inMemberType, outMemberType)
+				sw.Do("// FIXME: Provide conversion function to convert $.inType|raw$ to $.outType|raw$\n", args)
```

Comment:
I don't think this is a strong enough marker.  I would like compilation of the generated code to fail so that these problems are caught quickly.  I think we're going to hit at least two in openshift and I doubt we'd find it easily without a compilation error.

Reply:
I added:
sw.Do("compileErrorOnMissingConversion()\n")

in all places which will throw you a compile error.
Does that address your concern?